src/views/TemplateDetails/index.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="wpp_template_details">
+    <!-- Header -->
+    <div class="wpp_template_details__header">
+      <div class="wpp_template_details__header__title">
+        <div class="wpp_template_details__header__title__text">{{ selectedTemplate.name }}</div>
+        <div class="wpp_template_details__header__about__title__tag">
+          <unnnic-tag
+            scheme="neutral-dark"
+            :text="$t('WhatsApp.template_details.summary.marketing')"
```

Comment:
This text should not be static, you need to get the `selectedTemplate` category and display the translated corresponding label here